src/stylesheets/components/_alerts.scss

```diff
@@ -8,36 +44,32 @@ $alert-icons: (
   emergency: "error",
 );
 
-$alert-padding-left: units($theme-alert-padding-x) +
-  units($theme-alert-bar-width);
+$icon-object: (
+  "name": "ICON_NAME",
+  "svg-height": 40,
+  "svg-width": 40,
+  "height": $theme-icon-image-size,
+  "color": "ink",
+);
+
+$alert-padding-left: units($theme-alert-bar-width);
 
 .usa-alert {
   @include typeset($theme-alert-font-family);
   @include border-box-sizing;
-  background-color: color("base-lightest");
-  background-position: $alert-padding-left units($theme-alert-padding-x - 1);
-  background-repeat: no-repeat;
-  background-size: units($theme-alert-icon-size);
-  padding-bottom: units(2);
-  padding-left: $alert-padding-left;
-  padding-right: units($theme-alert-padding-x);
-  padding-top: units($theme-alert-padding-x);
+  @include set-text-and-bg(
+    "base-lightest",
+    $theme-alert-text-color,
+    $theme-alert-text-reverse-color
+  );
+  @include u-padding-y($theme-alert-padding-y);
   position: relative;
 
   * + & {
     margin-top: units(2);
   }
 
-  // TODO: why is this not simply a border?
-  &::before {
-    background-color: color("base-light");
-    content: "";
-    height: 100%;
-    left: 0;
-    position: absolute;
-    top: 0;
-    width: units($theme-alert-bar-width);
-  }
+  border-left: units($theme-alert-bar-width) solid color("base-light");
```

Comment:
Change alert bar to a border for simplicity

Reply:
We should move this up before `position: relative`.